docs/pages/inboxes/create-conversations.md

```diff
@@ -133,6 +133,47 @@ let group = try await alix.conversations.newGroup([bo.inboxId, caro.inboxId],
 
 :::
 
+## Optimistically create a group chat
+
+Use this method to optimistically create a group chat, which enables a user to create a group chat now and add members later.
+
+The group chat is created with just a name and is stored only in the app's local storage. The group chat is visible only to the creator in the app installation they used to create it.
```

Comment:
> The group chat is created with just a name

the group can also be created with any number of standard options, or no options.